Sources/MongoSwiftSync/Exports.swift

```diff
@@ -1,4 +1,4 @@
-// Generated using Sourcery 1.6.0 — https://github.com/krzysztofzablocki/Sourcery
+// Generated using Sourcery 1.6.1 — https://github.com/krzysztofzablocki/Sourcery
```

Comment:
do we need this change?

Reply:
yeah, that is because you have a newer version of sourcery locally than when this was generated. 
I was asking since it seems like running `make exports` isn't actually needed for this PR given we aren't exporting anything new and the content of this file isn't otherwise changing. generally, we try to keep unrelated/unnecessary changes in PRs to a minimum to cut down on noise and make it easier to trace the history of when things changed and why that said, this version is getting updated in your other PR anyway due to adding `ListIndexesOptions` so we can just leave it.